library/cartridge_get_facts_for_machines.py

```diff
@@ -41,7 +43,8 @@ def get_facts_for_machines(params):
 
         # Calculate single not expelled instance for each machine
         if all([
-            not helpers.is_expelled(instance_vars),
+            helpers.is_enabled(instance_vars),
+            instance_name not in cluster_disabled_instances,
```

Comment:
What about `helpers.is_enabled(instance_vars, cluster_disabled_instances)`? I've seen this code before.

Reply:
We should pass instance_name too. So, it will be strange: helpers.is_enabled(instance_vars, cluster_disabled_instances, instance_name)